dev/tools/localization/bin/gen_localizations.dart

```diff
@@ -543,19 +543,19 @@ void main(List<String> rawArgs) {
 
   // Maps of locales to resource key/value pairs for Widgets ARBs.
   final Map<LocaleInfo, Map<String, String>> widgetsLocaleToResources = <LocaleInfo, Map<String, String>>{};
-  // Maps of locales to resource key/attributes pairs for Widgets ARBs..
+  // Maps of locales to resource key/attributes pairs for Widgets ARBs.
   // https://github.com/googlei18n/app-resource-bundle/wiki/ApplicationResourceBundleSpecification#resource-attributes
   final Map<LocaleInfo, Map<String, dynamic>> widgetsLocaleToResourceAttributes = <LocaleInfo, Map<String, dynamic>>{};
 
   // Maps of locales to resource key/value pairs for Material ARBs.
   final Map<LocaleInfo, Map<String, String>> materialLocaleToResources = <LocaleInfo, Map<String, String>>{};
-  // Maps of locales to resource key/attributes pairs for Material ARBs..
+  // Maps of locales to resource key/attributes pairs for Material ARBs.
   // https://github.com/googlei18n/app-resource-bundle/wiki/ApplicationResourceBundleSpecification#resource-attributes
   final Map<LocaleInfo, Map<String, dynamic>> materialLocaleToResourceAttributes = <LocaleInfo, Map<String, dynamic>>{};
 
   // Maps of locales to resource key/value pairs for Cupertino ARBs.
   final Map<LocaleInfo, Map<String, String>> cupertinoLocaleToResources = <LocaleInfo, Map<String, String>>{};
-  // Maps of locales to resource key/attributes pairs for Cupertino ARBs..
+  // Maps of locales to resource key/attributes pairs for Cupertino ARB.
```

Comment:
Is the change from plural ARBs to singular ARB intentional? Is Cupertino special that way compared to Material and Widgets above?

Reply:
no that was just a typo, fixed